tests/manage/test_pvresize_with_node_down.py

```diff
@@ -0,0 +1,213 @@
+    pod_data['metadata']['name'] = f'{OCS_BUG_ID}-pod'
+    first_claim = pod_data['spec']['volumes'][0]
+    first_claim['persistentVolumeClaim']['claimName'] = pvc_name
+    first_container = pod_data['spec']['containers'][0]
+    first_volume_mount = first_container['volumeMounts'][0]
+    first_volume_mount['mountPath'] = f'/mnt/{OCS_BUG_ID}'
+
+    global POD
+    POD = Pod(**pod_data)
+    POD.create()
+
+    global CEPH_TOOL
+    CEPH_TOOL = get_ceph_tools_pod()
+
+
+def teardown():
+    """
+    Cleanup
+
+    Delete pod
+    Delete pvc
+    Delete namespace
+    Delete pv
+    """
+    pv_name = OUR_PVC.backed_pv
+    POD.delete()
+    POD.delete_temp_yaml_file()
+    OUR_PVC.delete()
+    OUR_PVC.delete_temp_yaml_file()
+    NAMESPACE.delete(resource_name=OCS_BUG_ID)
+    run_cmd(f'oc delete pv {pv_name}')
+
+
+def cmd_on_my_namespace(real_cmd):
+    """
+    Run an oc command on app pod in the test namespace
+
+    Retries if the command fails.
+
+    Raises:
+        CommandFailed if a failure limit is reached
+
+    Args:
+        real_cmd (str): oc command to be run ('get pods', for example)
+    """
+    fail_limit = 100
+    setup_cmd = f'rsh {OCS_BUG_ID}-pod {real_cmd}'
+    full_cmd = f'oc -n {config.ENV_DATA["my_namespace"]} {setup_cmd}'
+    not_run = True
+    fcount = 0
+    while not_run:
+        try:
+            run_cmd(full_cmd)
+            not_run = False
+        except CommandFailed as ex:
+            log.info(f'{ex} raised on {setup_cmd}')
+            fcount += 1
+            if fcount > fail_limit:
+                raise CommandFailed(f'Too many failures: {setup_cmd}')
+            sleep(5)
+
+
+def test_add_stuff_to_rbd():
+    """
+    Make sure that there is data in the rbd pool.
+
+    The /mnt/{OCS_BUG_ID} diretory on the app pod is mounted on a
+    block device that is implemented by openshift-storage rbd.
+    Writing here will add data to that storage.
+    """
+    df_info = CEPH_TOOL.exec_ceph_cmd('ceph df')
+    prev_avail = df_info['stats']['total_avail_bytes']
+    big_file = '/usr/lib/x86_64-linux-gnu/libicudata.so.57.1'
+    for i in range(0, 10):
+        param = f'cp {big_file} /mnt/{OCS_BUG_ID}/xx.{i}'
+        cmd_on_my_namespace(param)
+    df_info = CEPH_TOOL.exec_ceph_cmd('ceph df')
+    now_avail = df_info['stats']['total_avail_bytes']
+    assert now_avail < prev_avail
+
+
+def update_pvc():
+    """
+    Update the size of the pvc.
+
+    First modify the storage class to allow expansion.
+    Then read the pv so that we are sure that the change has taken effect
+    before increasing the size of the pvc
+    """
+    new_size = '200Gi'
+    cmd_front = f"oc -n {config.ENV_DATA['my_namespace']} patch"
+    cmd1_middle = 'storageclass/"rook-ceph-block"'
+    cmd2_middle = 'pvc/ocs-pvc'
+    cmd_1 = ' '.join([cmd_front, cmd1_middle, '--patch'])
+    cmd_1 += " '{\"allowVolumeExpansion\": true}'"
+    run_cmd(cmd_1)
+    run_cmd('oc get pv')
```

Comment:
As general note, if you need to use run_cmd, something is not good.
You should have all the classes to execute oc commands